backend/api/Controllers/InspectionController.cs

```diff
@@ -61,48 +60,19 @@ public async Task<ActionResult<TagInspectionMetadata>> Create([FromRoute] string
         /// </remarks>
         [HttpGet]
         [Authorize(Roles = Role.User)]
-        [Route("{installationCode}/{taskId}/taskId")]
+        [Route("{installationCode}/{isarInspectionId}/isarInspectionId")]
```

Comment:
Should we reference isar by name, or should we try to implicitly not couple isar and flotilla?

Reply:
I think it is no need to make it more general than we need to, it is fine to reference ISAR.

In the opposite way I agree, ISAR should not need to know about Flotilla; but Flotilla needs to know about ISAR, since it makes calls to it anyways.

Reply:
Since we have and inspectionId and an isarInspectionId, there is really no other way to reference the isar inspection Id, especially since it is generated by isar. We can have this discussion another time, but not relevant for this PR i think